modules/nodes-about-bare-metal-hosts-and-nodes.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// nodes/nodes/nodes-managing-bare-metal-nodes.adoc
+
+[id="nodes-about-bare-metal-hosts-and-nodes_{context}"]
+= About bare metal hosts and nodes
+
+In order to provision a bare metal host as a node in your cluster, you must first create a `machineset` CR object that corresponds to the bare metal host hardware. Bare metal host machine sets describe infrastructure components specific to your configuration. You apply specific Kubernetes labels to these machine sets and then update the infrastructure components to run on only those machines.
+
+When a `machineset` CR has been created for your hardware host, then create a `machine` CR object. `machine` CR's are used by {product-title} to create the bare metal node that corresponds to the host as specified in the `machineset` CR. Bare metal nodes are created automatically from the relevant `machine` CR.
+
+In the web console, you can navigate back and forth between the overview pages for the bare metal node and the parent bare metal host. features of bare metal nodes like network interfaces, hard disks, and host CPUs are displayed for bare metal nodes. The details of the parent bare metal host are also shown.
```

Comment:
```suggestion
In the web console, you can navigate back and forth between the overview pages for the bare metal node and the underlying bare metal host. Features of bare metal nodes like network interfaces, hard disks, and host CPUs are displayed for bare metal nodes. The details of the underlying bare metal host are also shown.
```

I believe this is a more accurate mental model. Nodes are Kube lingo for a place where workloads can run (at a technical level, a `kubelet`) and a host can be turned into a node by binding with a Machine CR.

Here's a crude diagram I once drew. 🙂

![image](https://user-images.githubusercontent.com/9122899/114251927-da224d00-9970-11eb-9bb2-a3f96d72ff06.png)

Reply:
Thanks! bookmarking this.